### PR TITLE
Testing

### PR DESCRIPTION
**Pull request by ericgaspar**
## Problem

- *Description of why you made this PR*

## Solution

- *And how do you fix that problem*

## PR Status

- [ ] Code finished and ready to be reviewed/tested
- [ ] The fix/enhancement were manually tested (if applicable)

## Automatic tests

Automatic tests can be triggered on https://ci-apps-dev.yunohost.org/ *after creating the PR*, by commenting "!testme", "!gogogadgetoci" or "By the power of systemd, I invoke The Great App CI to test this Pull Request!". (N.B. : for this to work you need to be a member of the Yunohost-Apps organization)

### CODE REVIEW

**Comment by ericgaspar** [comment]
!testme

**Comment by ericgaspar** [comment]
!testme

File: conf/config.php [hunk 1]
```diff
@@ -1,47 +1,36 @@
...
-	// mcrypt is still available.
-
-	// Key used for encryption of passwords for password-protected feeds
-	// in the database. A string of 24 random characters. If left blank, encryption
-	// is not used. Requires mcrypt functions.
-	// Warning: changing this key will make your stored feed passwords impossible
-	// to decrypt.
-	
-	define('SINGLE_USER_MODE', false);
+	putenv('TTRSS_SINGLE_USER_MODE=true');
```
Comment (zamentur) [inline conf/config.php:28]: Seems very strange to activate that ?
What happens for instances with multiple user ?

Comment (ericgaspar) [reply conf/config.php]: Patches that allows to connect to the app without authentication are not working anymore.
I see 3 possible solutions:

-  rework patches to work with upstream changes
-  use built-in authentication
-  set authentication to false and protect app access with SSO (as in this PR)

I am unable to fix the current patchs to the upstream changes.

**Review by alexAubin — APPROVED** [review]
Quick review, LGTM

What happend to the discussion regarding auth / singleuser mode that you were discussing in https://github.com/YunoHost-Apps/ttrss_ynh/pull/109#discussion_r776761168 in the end ?

**Comment by ericgaspar** [comment]
ttrss uses a built-in authentication. This used to be bypassed by a (not-working-anymore) patch.
It would be nice if someone familiar with PHP could adapt the patch to the new code ...

**Comment by ericgaspar** [comment]
Shall we merge this?

**Comment by ericgaspar** [comment]
!testme

**Pull request merged**